pyproject.toml

```diff
@@ -37,7 +37,7 @@ classic = [
     "chess==1.7.0",
     "rlcard==1.0.5",
     "pygame==2.1.3.dev8",
-    "hanabi_learning_environment==0.0.4",
+    "shimmy[openspiel]>=0.2.0"
```

Comment:
What do we think about pettingzoo depending on shimmy and shimmy depending on pettingzoo with this?

Reply:
So I talked a bit with mark about this I believe and the only problem is if there is a base requirement with each repo, leading to a circular import, so gymnasium has shimmy as an an extra/optional requirement to avoid that issue, this is doing the same thing. Should get confirmation from someone else though just to be sure this isn’t an issue.